composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
@xtreamwayz Please sort packages in alphabetical order as I requested in first comment. Thx!